plugins/permission-backend/src/service/router.ts

```diff
@@ -183,34 +237,56 @@ export async function createRouter(
   router.post(
     '/authorize',
```

Comment:
If you want to minimize the impact of this change it'd be better to introduce two new endpoints and deprecate this one, and same with the TS API. Still experimental so I think immediate breaking change makes sense and saves some time. I'm thinking I'd be best to clearly call out that all consumers of the API will be broken until both consumers and the permission backend have been redeployed

Reply:
Where specifically are you thinking we should call this out? There's already some stuff in the changesets, but perhaps we can make it stronger.

Reply:
I think early up in the permission backend breaking change callout, just noting being careful about rollout